src/webgpu/shader/execution/builtin/inversesqrt.spec.ts

```diff
@@ -0,0 +1,111 @@
+export const description = `
+Execution Tests for the 'inverseSqrt' builtin function
+`;
+
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { assert } from '../../../../common/util/util.js';
+import { GPUTest } from '../../../gpu_test.js';
+import { f32, f32Bits, Scalar, TypeF32, u32 } from '../../../util/conversion.js';
+
+import { Case, Config, kBit, run, ulpThreshold } from './builtin.js';
+
+export const g = makeTestGroup(GPUTest);
+
+// Calculates the linear interpolation between two values of a given ratio
+function lerp(min: number, max: number, t: number) {
+  if (!Number.isFinite(min) || !Number.isFinite(max) || !Number.isFinite(t)) {
+    return Number.NaN;
+  }
+  if (min === max) {
+    return min;
+  }
+
+  if (min > max) {
+    const temp = min;
+    min = max;
+    max = temp;
+  }
+
+  if (t > 1) {
+    return max;
+  }
+
+  if (t < 0) {
+    return min;
+  }
+
+  return min + t * (max - min);
+}
+
+// Unwrap Scalar params into numbers and test preconditions
+function unwrapRangeParams(min: Scalar, max: Scalar, steps: Scalar) {
```

Comment:
What is the steps parameter? The size of the step between min and max, or the number of steps.

I have to read the body to be sure.